manifests/m/Microsoft/Office/16.0.16227.20094/Microsoft.Office.yaml

```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
 
 PackageIdentifier: Microsoft.Office
-PackageVersion: 16.0.16130.20218
+PackageVersion: 16.0.16227.20094
```

Comment:
```suggestion
PackageVersion: 16.0.16130.20306
```

Reply:
In fact, I use this installer and a custom config file to install O365HomePremRetail and the version number here is the version number of retail version. Should we split it into Microsoft.Office.Enterprise and Microsoft.Office.Consumer?

Reply:
With the current installer switches and the default installation process, users would get the version suggested no? 👀 And the custom config scenario you're talking about - does it take other switches than the ones described in the manifest? If it doesn't I don't see how we can split the same installer into two packages

Reply:
It depends on the install configure file used. For example, I use `winget install Microsoft.Office --override '/configure https://gist.github.com/lychichem/15cff0e449c46262fb74218834324234/raw/25258c01f7a11482f30cfdc310a21c7fd946c666/Office365HomePrem.xml'` and it will install part of O365HomePremRetail (only Word, Excel, PowerPoint and OneNote installed). By passing different configuration file to the installer, SKU and components installed can be different.